app/Models/Setting.php

```diff
@@ -215,6 +216,40 @@ public static function passwordComplexityRulesSaving($action = 'update')
 
     }
 
-
-
+    /**
+     * Get the specific LDAP settings
+     * 
+     * @author Wes Hulette <jwhulette@gmail.com>
+     * 
+     * @since 5.0.0
+     *
+     * @return Collection
+     */
+    public static function getLdapSettings(): Collection
+    {
+        $ldapSettings = self::select([
```

Comment:
If you wanted to lose an abstraction layer, it could make sense to make this some kind of map between the Snipe-IT configuration setting names and the LdapAd object expected attribute names, perhaps?

Reply:
I was thinking that but I did the separate LDAP setting call so it very apparent what fields are needed and to easily add additional one if needed.